integration_tests/test_gov.py

```diff
@@ -18,7 +18,7 @@ def test_param_proposal(cluster, vote_option):
     """
```

Comment:
one TODO to have tests for the new gov proposals?

Reply:
opened: https://github.com/crypto-org-chain/chain-main/issues/835